CMakeLists.txt

```diff
@@ -156,12 +159,12 @@ endif()
 
 set(NOVELRT_LIBRARY_HEADERS
```

Comment:
You should add a `source_group(TREE ${CMAKE_CURRENT_SOURCE_DIR} FILES ${NOVELRT_LIBRARY_HEADERS})` after this `set` group.

It will cause the generated VS solution to have a nice directory structure (mirroring the on disk one), rather than pooling everything into one `Include Files` folder

Reply:
Similar for the other groups (`NOVELRT_LIBRARY_SOURCES` and the corresponding `NOVELRT_EXECUTABLE_*` ones)